tskv/src/kvcore.rs

```diff
@@ -475,3 +379,86 @@ impl TsKv {
         Ok(None)
     }
 }
+
+
+async fn run_wal_job(wal_opt: Arc<WalConfig>, mut receiver: UnboundedReceiver<WalTask>) -> Result<()> {
```

Comment:
If there is no refactoring， in place to modify

Reply:
change to function, remove &self